include/util/PrimalityTest.h

```diff
@@ -1,5 +1,13 @@
 #include "gmp.h"
+#include "util/Validation.h"
 
 int primaltyTest_millerrabin_mpz(const mpz_srcptr n, const int reps);
 
-int primaltyTest_millerrabin(const mpz_srcptr n, const mpz_srcptr nm1, const mpz_ptr x, const mpz_ptr y,const  mpz_srcptr q, const unsigned long int k);
\ No newline at end of file
+int primaltyTest_millerrabin(const mpz_srcptr n, const mpz_srcptr nm1, const mpz_ptr x, const mpz_ptr y,const  mpz_srcptr q, const unsigned long int k);
```

Comment:
Two things:
  * There is a typo in the function name: prima**lt**y instead of prima**li**ty,
  * Are there other usages of this function than those in `primaltyTest_isProbablePrime` (typo again)? If not, then we might consider making this function a static one inside the `.c` file.

Reply:
You are right, it is not necessary to keep them in the header.